packages/api/tests/endpoints/test-granules.js

```diff
@@ -251,6 +262,8 @@ test.beforeEach(async (t) => {
         granule_id: granuleId1,
         status: 'completed',
         collection_cumulus_id: t.context.collectionCumulusId,
+        published: true,
```

Comment:
(probably) minor: Was this set to true for the unpublish related action only?    Should it be for most tests?

Reply:
Yeah, it was explicitly for the test:

>remove a granule from CMR with launchpad authentication

That test uses the existing `fakePGGranules`. It could be refactored to create a new published granule (there's another test here that already does that) but I figured it might be good to have one of the granules we're using for _all_ of these tests set to published. It seemed like having a combination of published/unpublished granules might be more realistic?

Happy to just create new ones in the test if you think that's better - I just like having at least a little variety in the test fixtures when we can.

Reply:
I don't think it needs changing, just wanted to make sure I understood the reason for the change.